engineering/workflow.md

```diff
@@ -54,31 +55,153 @@ The team conducts a Sprint Planning meeting for 60 minutes at the beginning of e
+
+On rare occasions, when the team experiences duress or unpredictable and disruptive events, they may choose to have a specific retrospective to learn from those events.
+
+% TODO: Move the meeting into the team calendar for visibility: https://calendar.google.com/calendar/embed?src=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com
+
+#### The Roster for Facilitating Retrospectives
+
+There is a [Google Sheet](https://docs.google.com/spreadsheets/d/1s0ZSAxwFzJ-_WgDkZeicfuUadSFEjkjS6ZcNk4N7Mmg/edit?usp=sharing) in the shared team drive that determines who will be facilitating each retrospective meeting (as well as [sprint planning](meetings:sprint-planning) and backlog refinement meetings).
+Members of the engineering team are expected to self-nominate for this role because it is their improvement process.
+
+#### Retrospective Tool
+
+The team uses an [EasyRetro board](https://easyretro.io/publicboard/A8bu34hcK2eyg0s5MxNX0AfQXG02/7e47da43-a12b-4d21-842d-22361b799a92) to collect cards representing feedback concerning the last sprint.
+There are columns for:
+
+- Thanks and Celebrations
+- Things that Went Well
+- Things To Improve
+- Actions (Items to take forward into the next sprint).
+
+This template can be changed by tne facilitator. There are [many](https://retromat.org/) different template formats and the facilitator should choose one that is most appropriate for the team's need.
+
+### EasyRetro user account
+
+We have a paid Team subscription for EasyRetro, to make it easier to facilitate retrospectives. The user account is **admin@2i2c.org**. The credentials are stored in our [shared BitWarden account](https://vault.bitwarden.com/#/vault?organizationId=11313781-4b83-41a3-9d35-afe200c8e9f1).
+
+
+#### The General Format of a Iteartion Retrospective
+
+The retrospective meeting follows the below outline.
+
+1. _Identify a facilitator._
+
+The facilitator has the responsibility to:
+
+    * Ensure that their is psychological safety in the meeting.
+    * Ensure that the meeting isn't used as a blaming/ranting/finger pointing session.
+    * Share the [Retrospective Prime directive](https://retrospectivewiki.org/index.php?title=The_Prime_Directive) with the participants.
+
+1. _Set the context for the Iteration retrospective._
+
+   This involves explaining the period under observation, which process(es) we are trying to improve and what has been achieved by the current process. This could involve reviewing the 'Done' column in [](coordination:sprint-board).
+
+1. _Review the previous retrospective actions._
+
+    Reviewing previous retrospective actions involves:
+
+    * checking if the team completed all of the previous actions that they committed to
+    * noting how many actions were completed
+    * briefly discussing if the action has had the intended impact.
+
+   Retrospective actions are stored in the team's Sprint board and are tagged with the `Retro action` label.
```

Comment:
@haroldcampbell @consideRatio can I confirm my understanding:

By the time that we're reviewing retrospective actions, we're only looking at _committed_ actions (i.e. actions that were created in the previous sprint, and then refined and committed to in the iteration planning meeting. 

As such, by the time of the _next_ retrospective, we should have finished all of those actions i.e. there's no need to "drop them" here.

To be clear, that's what has been written here, I just wanted to confirm.

Reply:
Yes. Ideally, by the time of the next retrospective, we should have finished those actions. However, it's not a Black and White situation. There are going to be those actions that will (because of their nature) take more than a single iteration.

The key thing that I'm trying to communicate is that:

- Retro actions should be treated like work, because they are.
- Retro actions may need refining and will definitely need to meet the team's definition of ready. 
- Retro actions should make there way into the team's committed queue via Iteration Planning. The exceptions I think is when someone outside of the team is going to implement the actions (e.g. myself or Damián). In thoses cases, we'll still be governed by a WIP limit from either the Initiatives or Operations board.

Reply:
That matches my internal thinking exactly.

Reply:
I'm aligned with thinking above - but I didn't translate that clearly to the commented section. I'll review the section again and mark this as resolved.